README.md

```diff
@@ -29,7 +29,26 @@ compile 'com.bandwidth.sdk:messaging:(put desired version here)'
 public class MyAwesomeBandwidthMessagingApp{
     
     public static void main(String[] args){
+        MessagingClient client = new MessagingClient(
+                "u-aeawj73oafil", // your user id
+                "t-ayu44kfjhbf", // your api token
+                "soykuhkfalkjdf" // your api secret
+        );
+
+        Message message = client.sendMessage(SendMessageRequest.builder()
+                .from("+12223334444")
+                .addTo("+13334445555")
+                .addTo("+14445556666")
+                .applicationId("a-a7o34uhflaifadsf")
+                .text("This is a test group MMS message")
+                .addMedia("http://example.com/MyMedia.jpg")
+                .addMedia("http://example.com/OtherMedia.png")
+                .tag("An arbitrary value I will receive in associated callbacks")
+                .build()
```

Comment:
May not hurt to add some comments to these lines, like the addMedia line saying "Adding media is optional".